.vscode/tasks.json

```diff
@@ -87,6 +87,22 @@
       },
       "problemMatcher": []
     },
+    {
+      "label": "Update syrupy snapshots",
+      "detail": "Update syrupy snapshots for a given integration.",
+      "type": "shell",
+      "command": "python3 -m pytest ./tests/components/${input:integrationName} --snapshot-update -n auto",
```

Comment:
I'm not sure I'd add `-n auto`

Using `snapshot-update` in conjuction with `xdist` can makes it quite slow as it creates a temporary dump for each processor, and merges the dumps when all the tests have finalised

cc @jbouwh

Reply:
as smaller as the test suite for an integration is, as much worse get the overhead of `xdist` (_collecting items, cut the batches, create the workers, run the tests, combine the results, ..._)
so yeah, maybe it is overall better to let the `xdist` out for this task?

Reply:
i've just adopted it from the "code coverage" task where we also use `xdist`

Reply:
xdist is used in "code coverage" to better match CI, and highlight "leaking" tests early
I would keep it there, just remove it from the new task.